--- source/Makefile.old	2012-11-06 20:35:44.000000000 +0100
+++ source/Makefile	2012-11-06 20:46:49.000000000 +0100
@@ -18,10 +18,10 @@
 #
 # ----------------------------------------------------------------------------
 
-
-PREFIX = /usr/local
+CXX ?= g++
+PREFIX ?= /usr/local
 SUFFIX := $(shell uname -m | sed -e 's/^unknown/$//' -e 's/^i.86/$//' -e 's/^x86_64/$/64/')
-LIBDIR = lib$(SUFFIX)
+LIBDIR ?= lib$(SUFFIX)
 SHARED = $(PREFIX)/share/ambdec
 VERSION = 0.5.1
 CPPFLAGS +=  -ffast-math -Wall -MMD -MP -DVERSION=\"$(VERSION)\" -DSHARED=\"$(SHARED)\"
@@ -39,7 +39,7 @@
 ambdec:	LDLIBS += -lclxclient -lclthreads -ljack -lpng -lXft -lX11 -lrt
 ambdec:	LDFLAGS += -L/usr/X11R6/lib
 ambdec:	$(AMBDEC_O)
-	g++ $(LDFLAGS) -o $@ $(AMBDEC_O) $(LDLIBS)
+	$(CXX) $(LDFLAGS) -o $@ $(AMBDEC_O) $(LDLIBS)
 
 $(AMBDEC_O):
 -include $(AMBDEC_O:%.o=%.d)
@@ -50,7 +50,7 @@
 
 ambdec_cli:	LDLIBS += -lclthreads -ljack -lrt
 ambdec_cli:	$(AMBDEC_CLI_O)
-	g++ $(LDFLAGS) -o $@ $(AMBDEC_CLI_O) $(LDLIBS)
+	$(CXX) $(LDFLAGS) -o $@ $(AMBDEC_CLI_O) $(LDLIBS)
 
 $(AMBDEC_CLI_O):
 -include $(AMBDEC_CLI_O:%.o=%.d)
