diff -Naur rapicorn-17.0.0.orig/ui/sinfex.l rapicorn-17.0.0/ui/sinfex.l
--- rapicorn-17.0.0.orig/ui/sinfex.l	2018-04-03 11:05:00.236847732 -0500
+++ rapicorn-17.0.0/ui/sinfex.l	2018-04-03 11:05:22.253849858 -0500
@@ -2,7 +2,7 @@
  *                                                	-*-mode:c++;-*-
  */
 %option 8bit reentrant stack noyywrap yylineno
-%option ansi-definitions ansi-prototypes nounistd
+%option nounistd
 %option bison-bridge bison-locations
 %option nodefault warn
 %option prefix="flex_yy"
